notify.sh

```diff
@@ -5,7 +5,13 @@ SCRIPT_DIR=`dirname $0`
 
 PUSHOVER_TITLE="$1"
 PUSHOVER_MESSAGE="$2"
+FILE_TO_SEND="$3"
 PUSHOVER_URL="https://api.pushover.net/1/messages.json"
 
 ${CURL} -s -F "token=${PUSHOVER_TOKEN_APP}" -F "user=${PUSHOVER_TOKEN_USER}" -F "title=${PUSHOVER_TITLE}" -F "message=${PUSHOVER_MESSAGE}" ${PUSHOVER_URL} >> ${LOGFILE} 2>&1
+
+if [ $USE_TELEGRAM = "YES" ]; then
+    ${TELEGRAM} -W ${TELEGRAM_SERVER_KEY} -e "send_text $TELEGRAM_ACCOUNT_NAME $FILE_TO_SEND" >> ${LOGFILE} 2>&1
```

Comment:
Why you are using FILE_TO_SEND here and not the var PUSHOVER_MESSAGE? FILE_TO_SEND is the logfile (see your change in tracer.sh) - it it is getting bigger and bigger every time it is executed. This means I get every notification the whole logfile as a message?